tests/dev/script-create-population.R

```diff
@@ -0,0 +1,36 @@
+# Create Individual example.
+# Assuming PKSim installed
+
+initPKSim("C:/projects/PK-Sim/src/PKSim.R/bin/Debug/net472")
+
+moleculeOntogeny <- MoleculeOntogeny$new()
+moleculeOntogeny$molecule <- "MyMolecule"
+moleculeOntogeny$ontogeny <- "CYP3A4"
+
+dog <- createPopulationCharacteristics(
+  species = Species$Dog,
+  numberOfIndividuals = 50
+)
+
+print(dog)
+populationDog <- createPopulation(populationCharacteristics = dog)
+
+print(populationDog)
+
+human <- createPopulationCharacteristics(
+  species = Species$Human,
+  population = HumanPopulation$Asian_Tanaka_1996,
+  numberOfIndividuals = 50,
+  proportionOfFemales = 100,
+  ageMin = 20,
+  ageMax = 30,
+  weightMin = 50,
+  moleculeOntogenies = moleculeOntogeny
+)
+
+print(human)
+
+populationHuman <- createPopulation(populationCharacteristics = human)
+print(populationHuman)
+
+populationHuman$getParameterValues("Organism|Age")

```

Comment:
please also add a test for Preterm-Pop-Creation

Reply:
These are no tests. Just local script for the dev to play with.

There are in fact no unit tests for create individual and population at all because it requires Pksim to be installed. Something we might have to do but it will require messing around with the build script